CRM/Member/BAO/Membership.php

```diff
@@ -1164,10 +1164,8 @@ public static function fixMembershipStatusBeforeRenew(&$currentMembership, $chan
       $currentMembership
     );
 
-    if (empty($status) ||
-      empty($status['id'])
-    ) {
-      CRM_Core_Error::fatal(ts('Oops, it looks like there is no valid membership status corresponding to the membership start and end dates for this membership. Contact the site administrator for assistance.'));
+    if (empty($status) || empty($status['id'])) {
+      throw new CRM_Core_Exception(ts('Oops, it looks like there is no valid membership status corresponding to the membership start and end dates for this membership. Contact the site administrator for assistance.'));
```

Comment:
@mattwire wanna ping this one liner in a separate PR & I'll merge that

Reply:
#13774